src/configReader.ts

```diff
@@ -0,0 +1,20 @@
+import * as fs from "fs";
+import * as path from "path";
+
+export class ConfigReader {
+    rootDir: string;
+
+    constructor(rootDir: string) {
+        this.rootDir = rootDir;
+    }
+
+    readConfigFile(...filePath: string[]): object | null {
```

Comment:
if the function / class is for reading config files, should it be allowed to have parameters? I think the filePath could be something known by ConfigReader. are we expecting there to be other config files to read?